interactive_continue_printer.go

```diff
@@ -0,0 +1,183 @@
+package pterm
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"atomicgo.dev/cursor"
+	"atomicgo.dev/keyboard"
+	"atomicgo.dev/keyboard/keys"
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
+)
+
+var (
+	// DefaultInteractiveContinue is the default InteractiveContinue printer.
+	// Pressing "y" will return yes, "n" will return no, "a" returns all and "s" returns stop.
+	// Pressing enter without typing any letter will return the configured default value (by default set to "yes", the fisrt option).
+	DefaultInteractiveContinue = InteractiveContinuePrinter{
+		DefaultValueIndex: 0,
+		DefaultText:       "Do you want to continue",
+		TextStyle:         &ThemeDefault.PrimaryStyle,
+		Options:           []string{"yes", "no", "all", "stop"},
+		OptionsStyle:      &ThemeDefault.SuccessMessageStyle,
+		SuffixStyle:       &ThemeDefault.SecondaryStyle,
+	}
+)
+
+// InteractiveContinuePrinter is a printer for interactive continue prompts.
+type InteractiveContinuePrinter struct {
+	DefaultValueIndex int
+	DefaultText       string
+	TextStyle         *Style
+	Options           []string
+	OptionsStyle      *Style
+	Handles           []string
+	ShowshortHandles  bool
+	SuffixStyle       *Style
+}
+
+// WithDefaultText sets the default text.
+func (p InteractiveContinuePrinter) WithDefaultText(text string) *InteractiveContinuePrinter {
+	p.DefaultText = text
+	return &p
+}
+
+// WithDefaultValueIndex sets the default value, which will be returned when the user presses enter without typing any letter.
+func (p InteractiveContinuePrinter) WithDefaultValueIndex(value int) *InteractiveContinuePrinter {
+	if value >= len(p.Options) {
+		panic("Index out of range")
+	}
+	p.DefaultValueIndex = value
+	return &p
+}
+
+// WithDefaultValue sets the default value, which will be returned when the user presses enter without typing any letter.
+func (p InteractiveContinuePrinter) WithDefaultValue(value string) *InteractiveContinuePrinter {
+	for i, o := range p.Options {
+		if o == value {
+			p.DefaultValueIndex = i
+			break
+		}
+	}
+	return &p
+}
+
+// WithTextStyle sets the text style.
+func (p InteractiveContinuePrinter) WithTextStyle(style *Style) *InteractiveContinuePrinter {
+	p.TextStyle = style
+	return &p
+}
+
+// WithOptions sets the options.
+func (p InteractiveContinuePrinter) WithOptions(options []string) *InteractiveContinuePrinter {
+	p.Options = options
+	return &p
+}
+
+// WithHandles allows you to customize the short handles for the answers.
+func (p InteractiveContinuePrinter) WithHandles(handles []string) *InteractiveContinuePrinter {
+	if len(handles) != len(p.Options) {
+		panic("Invalid number of handles")
```

Comment:
Instead of panicking, could you please fall back to the same logic, as if there were no custom handles? Such mistakes make it to production fast, as many don't test the full CLI manually before deployment. That way, the users could still use it, even if it's configured wrong.

Reply:
I've updated the PR to follow your preference but I do think this will result in a harder to debug problem.